crates/bin/README.md

```diff
@@ -14,7 +14,7 @@ To include liquid in your project add the following to your Cargo.toml:
 
 ```toml
 [dependencies]
-liquid-compiler = "0.20"
+liquid-compiler = "0.25.0"
```

Comment:
To prevent this from happening again, could you modify each `Cargo.toml` to update this automatically?

To see how, look in `/Cargo.toml` for
```toml
[package.metadata.release]
pre-release-replacements = [
```

You can run `cargo release major --no-publish -vvv` and you should see diffs being shown of the replacements being applied.

If not, its still at least an improvement that these are updated and I can try to remember to update the replacements later.

Reply:
Yes, will do - thanks for the guidance!

Reply:
Actually - a question: should the `liquid-compiler =` in the `crates/bin/README.md` file be renamed to `liquid-bin =`? The names of all the others match up, but not this one.